integreat_cms/cms/utils/linkcheck_utils.py

```diff
@@ -229,7 +236,9 @@ def replace_links(search, replace, region=None, user=None, commit=True):
                 new_translation = deepcopy(translation)
                 for link in translation.links.select_related("url"):
                     url = link.url.url
-                    if search in url:
+                    if search in url and (
+                        not only_internal or link.url.type != "external"
```

Comment:
Since the scope of this issue changed a little bit (renaming regions is no longer the main use case), maybe allowing the replacement of external links as well isn't too bad? E.g. if we have many external links:

https://old-domain.tld/first-page
https://old-domain.tld/second-page

and they need to be replaced with:

https://new-domain.tld/first-page
https://new-domain.tld/second-page

But I wonder whether we should make this configurable by the user? E.g. a select field to limit the replacement to a specific link type, including an "all types" option to not limiting it?
Not sure though, I guess leaving it as it is for the moment is fine, and we can still think about extending it in the future...

Reply:
Choice of link type is now available. I hope this helps users to avoid replacing a lot of links accidentally (like, many links to the website of city office or health department get replaced together while users wanted to replace only internal links..)